test/checks/translation_key_exists_test.rb

```diff
@@ -39,6 +39,38 @@ def test_ignores_existing_key
     assert_offenses("", offenses)
   end
 
+  def test_ignores_key_included_in_schema
+    offenses = analyze_theme(
+      ThemeCheck::TranslationKeyExists.new,
+      "sections/product.liquid" => <<~END,
+        {% schema %}
+          {
+            "locales": {
+              "en": {
+                "submit": "Subscribe"
+              }
+            }
+          }
+        {% endschema %}
+        {{"submit" | t}}
```

Comment:
The schema is usually at the bottom of a file. Not the top. I believe this breaks if we flip the order.